core/examples/luigi-sample-angular/e2e/tests/context-switcher.spec.js

```diff
@@ -34,33 +18,97 @@ describe('Context switcher', () => {
       'contain',
       'Environment 1'
     );
+  });
+
+  it('Add and remove project with context switcher', () => {
+    cy.goToProjectsPage();
+
+    cy.get('.fd-app__sidebar').should('contain', 'Project One');
+    cy.get('.fd-app__sidebar').should('contain', 'Project Two');
+    cy.get('.fd-app__sidebar').should('not.contain', 'Project 3');
 
-    // Overview
     cy.goToOverviewPage();
+    cy.expectPathToBe('/overview');
 
     cy.get('[data-cy=luigi-alert]').should('not.exist');
 
-    // check label
-    cy.get('.fd-product-menu .fd-popover__control button').should(
-      'contain',
-      'Select Environment ...'
-    );
+    // add project
+
+    cy.selectContextSwitcherItem('New Project');
 
-    cy.get('.fd-product-menu')
-      .contains('Select Environment ...')
-      .click();
+    cy.expectPathToBe('/projects');
 
-    // click an action
-    cy.get('.fd-product-menu .fd-popover__body')
-      .contains('New Environment (bottom)')
-      .click();
+    cy.get('.fd-app__sidebar').should('contain', 'Project One');
+    cy.get('.fd-app__sidebar').should('contain', 'Project Two');
+    cy.get('.fd-app__sidebar').should('contain', 'Project 3');
 
     cy.get('[data-cy=luigi-alert]').should(
       'have.class',
       'fd-alert--information'
     );
 
-    cy.get('[data-cy=luigi-alert-dismiss]').click();
-    cy.get('[data-cy=luigi-alert]').should('not.exist');
+    cy.get('[data-cy=luigi-alert]').should('contain', 'Project 3 created.');
+
+    cy.goToOverviewPage();
+    cy.expectPathToBe('/overview');
+
+    // remove project
+
+    cy.selectContextSwitcherItem('Remove Project');
+
+    cy.expectPathToBe('/projects');
+
+    cy.get('.fd-app__sidebar').should('contain', 'Project One');
+    cy.get('.fd-app__sidebar').should('contain', 'Project Two');
+    cy.get('.fd-app__sidebar').should('not.contain', 'Project 3');
```

Comment:
I thought about that and in this case I wouldn't write a function, since the assertions are different in each case (_contain_ vs _not.contain_). Creating a function here would be overcomplicating it IMHO and making the test less readable.